extensions/wikia/Search/classes/Test/Result/ResultHelperTest.php

```diff
@@ -0,0 +1,9 @@
+<?php
+namespace Wikia\Search\Test\ResultSet;
+use \WikiaBaseTest as WikiaBaseTest, Wikia\Search\Result\ResultHelper;
+
+class AbstractResultSetTest extends WikiaBaseTest {
```

Comment:
The name needs to change to reflect what this test does AND prevent fatals during test execution